newsroom/commands/initialize_data.py

```diff
@@ -22,7 +24,10 @@ def run(self, entity_name=None, force=False, init_index_only=False):
         # create indexes in mongo
         app.init_indexes()
         # put mapping to elastic
-        app.data.init_elastic(app)
+        try:
+            app.data.init_elastic(app)
+        except elasticsearch.exceptions.TransportError as err:
+            logger.error("Error when initializing elastic %s", err)
```

Comment:
Any idea about this error:
```bash
Error when initializing elastic TransportError(500, 'mapper_exception', "Can't update attribute for type [_doc.extra.enabled] in index mapping")
```

Reply:
yep that's due to reusing previous index, needs reindexing, but first I need updated instance